common/generated_libs/network_protocol/network_protocol.pb.h

```diff
@@ -16,7 +16,8 @@ extern "C" {
 /* Enum definitions */
 typedef enum _VentMode {
     VentMode_OFF = 0,
-    VentMode_PRESSURE_CONTROL = 1
+    VentMode_PRESSURE_CONTROL = 1,
```

Comment:
Regenerating the code causes more changes: here's the diff I obtained after checking out your PR and regenerating it:

```
diff --git a/common/generated_libs/network_protocol/network_protocol.pb.h b/common/generated_libs/network_protocol/network_protocol.pb.h
index 976a96c..682b9fc 100644
--- a/common/generated_libs/network_protocol/network_protocol.pb.h
+++ b/common/generated_libs/network_protocol/network_protocol.pb.h
@@ -76,8 +76,8 @@ typedef struct _GuiStatus {
 
 /* Helper constants for enums */
 #define _VentMode_MIN VentMode_OFF
-#define _VentMode_MAX VentMode_PRESSURE_CONTROL
-#define _VentMode_ARRAYSIZE ((VentMode)(VentMode_PRESSURE_CONTROL+1))
+#define _VentMode_MAX VentMode_PRESSURE_ASSIST
+#define _VentMode_ARRAYSIZE ((VentMode)(VentMode_PRESSURE_ASSIST+1))
 
 #define _AlarmKind_MIN AlarmKind_RESPIRATORY_RATE_TOO_LOW
 #define _AlarmKind_MAX AlarmKind_TIDAL_VOLUME_TOO_HIGH
@@ -100,10 +100,10 @@ typedef struct _GuiStatus {
 #define Alarm_start_time_tag                     1
 #define Alarm_kind_tag                           2
 #define SensorReadings_patient_pressure_cm_h2o_tag 1
-#define SensorReadings_inflow_pressure_diff_cm_h2o_tag 4
-#define SensorReadings_outflow_pressure_diff_cm_h2o_tag 5
 #define SensorReadings_volume_ml_tag             2
 #define SensorReadings_flow_ml_per_min_tag       3
+#define SensorReadings_inflow_pressure_diff_cm_h2o_tag 4
+#define SensorReadings_outflow_pressure_diff_cm_h2o_tag 5
 #define VentParams_mode_tag                      1
 #define VentParams_peep_cm_h2o_tag               3
 #define VentParams_breaths_per_min_tag           4
```